test/services/android-debug-service.ts

```diff
@@ -0,0 +1,161 @@
+			},
+			{
+				scenarioName: "useBundledDevTools is true and useHttpUrl is false",
+				debugOptions: {
+					useBundledDevTools: true,
+					useHttpUrl: false
+				},
+				expectedChromeUrl: `chrome-devtools://devtools/bundled/inspector.html?experiments=true&ws=localhost:${expectedPort}`,
+			},
+			{
+				scenarioName: "useBundledDevTools is true and useHttpUrl is true",
+				debugOptions: {
+					useBundledDevTools: true,
+					useHttpUrl: true
+				},
+				expectedChromeUrl: `https://chrome-devtools-frontend.appspot.com/serve_file/@02e6bde1bbe34e43b309d4ef774b1168d25fd024/inspector.html?experiments=true&ws=localhost:${expectedPort}`,
+			},
+
+			// When useBundledDevTools is false
+			{
+				scenarioName: "useBundledDevTools is false and useHttpUrl is not passed",
+				debugOptions: {
+					useBundledDevTools: false
+				},
+				expectedChromeUrl: `chrome-devtools://devtools/remote/serve_file/@02e6bde1bbe34e43b309d4ef774b1168d25fd024/inspector.html?experiments=true&ws=localhost:${expectedPort}`,
+			},
+			{
+				scenarioName: "useBundledDevTools is false and useHttpUrl is false",
+				debugOptions: {
+					useBundledDevTools: false,
+					useHttpUrl: false
+				},
+				expectedChromeUrl: `chrome-devtools://devtools/remote/serve_file/@02e6bde1bbe34e43b309d4ef774b1168d25fd024/inspector.html?experiments=true&ws=localhost:${expectedPort}`,
+			},
+			{
+				scenarioName: "useBundledDevTools is false and useHttpUrl is true",
+				debugOptions: {
+					useBundledDevTools: false,
+					useHttpUrl: true
+				},
+				expectedChromeUrl: `https://chrome-devtools-frontend.appspot.com/serve_file/@02e6bde1bbe34e43b309d4ef774b1168d25fd024/inspector.html?experiments=true&ws=localhost:${expectedPort}`,
+			},
+
+			// When useBundledDevTools is not passed
+			{
+				scenarioName: "useBundledDevTools is not passed and useHttpUrl is false",
+				debugOptions: {
+					useHttpUrl: false
+				},
+				expectedChromeUrl: `chrome-devtools://devtools/bundled/inspector.html?experiments=true&ws=localhost:${expectedPort}`,
+			},
+			{
+				scenarioName: "useBundledDevTools is not passed and useHttpUrl is true",
+				debugOptions: {
+					useHttpUrl: true
+				},
+				expectedChromeUrl: `https://chrome-devtools-frontend.appspot.com/serve_file/@02e6bde1bbe34e43b309d4ef774b1168d25fd024/inspector.html?experiments=true&ws=localhost:${expectedPort}`,
+			},
+
+			// devToolsCommit tests
+
+			{
+				scenarioName: "devToolsCommit defaults to 02e6bde1bbe34e43b309d4ef774b1168d25fd024 when useBundledDevTools is set to false",
+				debugOptions: {
+					useBundledDevTools: false
+				},
+				expectedChromeUrl: `chrome-devtools://devtools/remote/serve_file/@02e6bde1bbe34e43b309d4ef774b1168d25fd024/inspector.html?experiments=true&ws=localhost:${expectedPort}`,
+			},
+			{
+				scenarioName: "devToolsCommit is disregarded when useBundledDevTools is not passed",
+				debugOptions: {},
```

Comment:
The idea was to keep a separate test case for this scenario. This way, we may change the test you've mentioned in the future, but this one should be kept this way as it's clear what is its purpose.